src/Corax/IndexWriter.cs

```diff
@@ -244,10 +254,11 @@ void AddMaybeAvoidDuplicate(List<long> term, long entryId)
                     break;
 
                 case IndexEntryFieldType.Tuple:
-                    if (entryReader.Read(binding.FieldId, out Span<byte> valueInEntry) == false)
+                    if (entryReader.Read(binding.FieldId, out _,  out long lVal, out double dVal,out Span<byte> valueInEntry) == false)
                         break;
 
                     ExactInsert(valueInEntry);
+                    NumericInsert(lVal, dVal);
                     break;
```

Comment:
Try `NumericInsert` on top instead, looks like something with the parser.

Reply:
I don't follow? 
This insert a numeric token